db/migrate/20160816000111_create_students.rb

```diff
@@ -0,0 +1,11 @@
+class CreateStudents < ActiveRecord::Migration
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :grade
+      t.integer :teacher_id
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
Great job! This is exactly what the student table should look like for now. But, we'll need a Student controller, go ahead and create an empty student controller and push it to this branch.